pytorch_transformers/tokenization_utils.py

```diff
@@ -696,15 +698,17 @@ def encode(self, text, text_pair=None, add_special_tokens=False, **kwargs):
         second_sentence_tokens = [self._convert_token_to_id(token) for token in self.tokenize(text_pair, **kwargs)]
 
         if add_special_tokens:
-            return self.add_special_tokens_sentences_pair(first_sentence_tokens, second_sentence_tokens)
+            return self.add_special_tokens_sentences_pair(first_sentence_tokens, second_sentence_tokens, output_mask)
         else:
+            if output_mask:
+                logger.warning("Can't output mask if no special tokens are involved. Please call the method with add_special_tokens set to True.")
             return first_sentence_tokens, second_sentence_tokens
 
     def add_special_tokens_single_sentence(self, token_ids):
         logger.warning("This tokenizer does not make use of special tokens. The sequence has been returned with no modification.")
         return token_ids
 
-    def add_special_tokens_sentences_pair(self, token_ids_0, token_ids_1):
+    def add_special_tokens_sentences_pair(self, token_ids_0, token_ids_1, output_mask=False):
```

Comment:
I think we should still output a mask here for consistency of the return type (tuple) across models. Maybe just a list of zeros.

Reply:
This would introduce a breaking change in the encoder method, wouldn't it?